### PR TITLE
GtkMinimal: Reenable sample

### PR DESCRIPTION
**Pull request by badcel**
Still work in progress

### CODE REVIEW

**Comment by firox263** [comment]
I know this is still WIP, but a heads-up about some of the constructors:

```cs
public Application(string applicationId, ApplicationFlags flags = ApplicationFlags.FlagsNone) 
    : this(Native.@new(applicationId, flags)) { }
```

From what I recall, the `this(Native.@new)` pattern breaks when subclassing, as we need to be going through either `Object()` or `Object(ConstructParameter[])` for the type integration code to run. The `IntPtr` constructor would simply wrap the `GtkApplication` as a foreign-created object.

**Comment by badcel** [comment]
The sample is working again. I think we can start with this simple Builder support, as it is not infiltrating our inheritance chain like the original concept. I hope to see the template support in the future.

File: Samples/Gtk3/GtkMinimal/DemoWindow.cs [hunk 1]
```diff
@@ -13,37 +13,41 @@ public class DemoWindow : ApplicationWindow
...
         {
+            Application = application;
+            builder.Connect(this);
+            
             // Connect Button
-            Button.Clicked += Button_Clicked;
+            Button.OnClicked += Button_Clicked;
 
             // Create Notebook
             notebook = new Notebook();
```
Comment (na2axl) [inline Samples/Gtk3/GtkMinimal/DemoWindow.cs:26]: @badcel can we use the object initializer (with signal indexers) here ?

**Review by na2axl — APPROVED** [review]
LGTM, only the little enhancement I think we can add in the DemoWindow.cs. 👍🏾

File: Samples/Gtk3/GtkMinimal/DemoWindow.cs (hunk 1 above)
Comment (badcel) [reply Samples/Gtk3/GtkMinimal/DemoWindow.cs:26]: Hm, I like to see that the "classic" style is possible, too. We should probably show the possiblities which are available.

I don't like how the samples are named, but this is for another PR.

**Pull request merged**